scm/webhook.go

```diff
@@ -37,6 +37,15 @@ type (
 		Commits []Commit
 	}
 
+	// PipelineHook
+	PipelineHook struct {
```

Comment:
most hooks generally have an `Action` that tells us the purpose of the hook (create, update, execution, etc)

Reply:
There is no such field in payload which tell us the purpose of the hook